Resources/views/Collector/icon.svg

```diff
@@ -0,0 +1,4 @@
+<svg version="1.1"xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
```

Comment:
plesae note that this is copied from the DoctrineBundle, so it's wrong there as well.